### PR TITLE
feature(unlock-app): Enhance dashboard header

### PR DESCRIPTION
**Pull request by 0xTxbi**
# Description
This PR introduces a standardized dashboard header to replace individual headers in events, subscriptions, and certifications, ensuring consistency and reducing redundancy.

# Issues
Fixes #
Refs #13673

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx [hunk 1]
```diff
@@ -38,8 +26,59 @@ export default function DashboardHeader({
...
+        url: '/certification',
+        src: '/images/svg/logo-unlock-certificate.svg',
+      }
+    }
+    return {
+      url: '/',
+      src: '/images/svg/unlock-logo.svg',
+    }
+  }
+  console.log('pathname', pathname)
```
Comment (julien51) [inline unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx:54]: 🧹

File: unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx [hunk 2]
```diff
@@ -38,8 +26,59 @@ export default function DashboardHeader({
...
+
+  // Determine menu sections based on pathname
+  const getMenuSections = () => {
+    if (pathname?.includes('event')) {
+      return [{ title: 'Events', url: '/my-events' }]
```
Comment (julien51) [inline unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx:59]: I actually think we should show the same menus everywhere...

File: unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx (hunk 1 above)
Comment (0xTxbi) [reply unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx:54]: dang! an oversight! thanks!

File: unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx (hunk 2 above)
Comment (0xTxbi) [reply unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx:59]: alright, got it!

**Review by julien51 — APPROVED** [review]

**Pull request merged**